packages/snake/src/snake.js

```diff
@@ -92,6 +93,10 @@ export function executeTick(board, isSnakeLengtheningTick) {
     }
     const collided = moveSnake(board.snake, board.snakeDirection, board.apples)
 
+    if (outOfBounds(board, snake) || conflictsWith(snake[0].x, snake[0].y, snake.slice(1))) {
```

Comment:
This is nitpicking, but I believe `conflictsWith` would be better with a signature of `conflictsWith(item, items)` rather than `conflictsWith(x, y, item)`